### PR TITLE
satellite/console: Add security headers

### PR DESCRIPTION
**Pull request by brimstone**
What: Adds X-Frame-Options and Referrer-Policy security headers to the
satellite console.

Why: There's a bunch of HTTP Headers that secure sites should use. This
PR adds two to the ones already sent.

Please describe the tests: Existing tests.

Please describe the performance impact: Maybe a little bit?

## Code Review Checklist (to be filled out by reviewer)
 - [x] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible?
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

File: satellite/console/consoleweb/server.go [hunk 1]
```diff
@@ -214,6 +214,8 @@ func (server *Server) appHandler(w http.ResponseWriter, r *http.Request) {
 	header.Set(contentType, "text/html; charset=UTF-8")
 	header.Set("Content-Security-Policy", strings.Join(cspValues, "; "))
 	header.Set("X-Content-Type-Options", "nosniff")
+	header.Set("X-Frame-Options", "DENY")        // Prevent being included in a <frame> or <iframe>
```
Comment (crawter) [inline satellite/console/consoleweb/server.go]: hm...but we have a universal login page that renders login and register pages in Iframe

Comment (brimstone) [reply satellite/console/consoleweb/server.go]: oh neat. I didn't know that was a thing. I updated it to support the tardigrade.io domain. We'll want to move this to a config setting at some point.

File: satellite/console/consoleweb/server.go [hunk 2]
```diff
@@ -206,14 +206,16 @@ func (server *Server) appHandler(w http.ResponseWriter, r *http.Request) {
 
 	cspValues := []string{
 		"default-src 'self'",
-		"script-src 'self' *.stripe.com cdn.segment.com",
+		"frame-ancestors tardigrade.io",
```
Comment (crawter) [inline satellite/console/consoleweb/server.go]: why should storj repo knows smth about tardigrade branding? Would it better to place this values in config?

Comment (crawter) [reply satellite/console/consoleweb/server.go]: actually, the same with segment and stripe...

File: satellite/console/consoleweb/server.go [hunk 3]
```diff
@@ -62,12 +62,13 @@ type Config struct {
 
 	PasswordCost int `internal:"true" help:"password hashing cost (0=automatic)" default:"0"`
 
+	ContactInfoURL        string `help:"url link to contacts page" default:"https://forum.storj.io"`
+	FrameAncestors        string `help:"allow domains to embed the satellite in a frame" default:"tardigrade.io"`
```
Comment (thepaul) [inline satellite/console/consoleweb/server.go]: Is it worth mentioning in the help text (or in a comment maybe) that multiple domains should be space-separated? I always hate having to go spelunking through code to figure out what an app expects

Comment (brimstone) [reply satellite/console/consoleweb/server.go]: I'm always up for better documentation.

File: satellite/console/consoleweb/server.go (hunk 2 above)
Comment (brimstone) [reply satellite/console/consoleweb/server.go]: I'll handle my addition. Those can be configured later.

**Pull request merged**